src/resolution_functions/instrument.py

```diff
@@ -738,6 +741,10 @@ def format_available_models_options(self) -> str:
             if isinstance(model_data, str):
                 contents.append([model_name, model_data, '', ''])
             else:
+                if not model_data['configurations']:
+                    contents.append([model_name, '', '', ''])
+                    continue
```

Comment:
Although I would probably suggest changing the order so the `if` check is positive.
```suggestion
            elif not model_data['configurations']:
                contents.append([model_name, '', '', ''])
            else:
```

Reply:
Good point, thanks! Though I prefer to have a short `elif` statement (even if it contains `not`) rather than having a large `elif` followed by a single-line `else`, so I'll go with the suggestion as-is